gcp_deepvariant_runner.py

```diff
@@ -277,6 +335,22 @@ def _gcs_object_exist(gcs_obj_path):
     return False
 
 
+def _get_gcs_object_size(gcs_obj_path):
+  """Returns the size of the given GCS object.
+
+  Args:
+    gcs_obj_path: (str) a path to an obj on GCS.
+  """
+  if not _gcs_object_exist(gcs_obj_path):
+    return 0
+  storage_client = storage.Client()
+  bucket_name = _get_gcs_bucket(gcs_obj_path)
+  obj_name = _get_gcs_relative_path(gcs_obj_path)
+  bucket = storage_client.get_bucket(bucket_name)
```

Comment:
Original comments

"
nmousavi@
Does this and the below throw exceptions? If yes either catch or declare in the comment.

@samanvp
Good point, I reuse the existing helper function.
"

Reply:
Did you change it?

Reply:
Ah, I see. This function is very similar to `_gcs_object_exist`. I would put shared pieces into `_get_gcs_obj()` and then call `.size()` and `.exist()` in these two methods.

Reply:
I got rid of redundant code by implementing `_gcs_object_exist()`simply by checking whether the size is != 0.
I think this is a better way to deal with all the complexity of exceptions raised by storage module and NoneType returned when blob is missing. Otherwise, we would have to repeat a check after each call of: 
```
obj = _get_gcs_obj(...)
if obj is None:
  ... 
else:
  ...
```
Also I found a bug in `_is_valid_gcs_path()`, currently it return True for `gs:/test-bucket/` (note missing '/') and this will cause a exception further down when it tries to parse the url.

Reply:
This is not correct, we can have an obj of size 0 on GCS. In other words, `size == 0` doesn't imply non-existence. 

Also, `_get_gcs_obj()` should not return an null obj, this is design.

Reply:
Discussed offline and we decided to remove `_gcs_object_exist()` method and only rely on `_get_gcs_object_size()` for our flag values validation.